package.json

```diff
@@ -129,7 +129,7 @@
     "extract-text-webpack-plugin": "2.0.0-beta.5",
     "file-loader": "0.9.0",
     "gemini": "4.16.0",
-    "gemini-browserstack": "1.0.0",
+    "gemini-browserstack": "https://github.com/HSLdevcom/gemini-browserstack",
```

Comment:
Required until https://github.com/Saulis/gemini-browserstack/pull/2 is merged and released